osler/core/api/serializers.py

```diff
@@ -23,18 +23,71 @@ class Meta(object):
 class PatientSerializer(serializers.ModelSerializer):
     class Meta(object):
         model = models.Patient
-        exclude = []
+        exclude = ['case_managers']
 
-    history = HistorySerializer()
-    latest_workup = WorkupSerializer()
-    gender = serializers.StringRelatedField(read_only=True)
+    # history = HistorySerializer()
+    # latest_workup = WorkupSerializer()
+    # gender = serializers.StringRelatedField(read_only=True)
```

Comment:
You can just delete old code. We're under version control, so we can get it back later